src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -9040,7 +9050,18 @@ class CL0<T>{}
 ", parseOptions: TestOptions.Regular8);
 
             c.VerifyDiagnostics(
-                );
+                // (13,24): warning CS8622: Nullability of reference types in type of parameter 'x' of 'CL0<string?> C.M1(CL0<string> x)' doesn't match the target delegate 'C.D1'.
```

Comment:
>// (13,24): warning CS8622: Nullability of reference types in type of parameter 'x' of 'CL0<string?> C.M1(CL0<string> x)'  [](start = 16, length = 122)

I believe all these warnings are unexpected because delegates are explicitly created. Should be treated as an explicit cast I believe.

Reply:
Reverted change.